module-apps/application-settings/windows/OptionsWidgetMaker.cpp

```diff
@@ -11,7 +11,7 @@ namespace gui
     {}
 
     void OptionsWidgetMaker::addSwitchOption(
-        std::list<gui::Option> &list, const UTF8 &text, bool &option, std::function<void()> callback, bool indent)
+        std::list<gui::Option> &list, UTF8 &&text, bool &option, const std::function<void()> &callback, bool indent)
```

Comment:
as for && can't we just remove std::move on the text below on `text`? and both: && and std::move wont be required :)

Reply:
Changed as you suggested ;)